Makefile

```diff
@@ -59,10 +59,10 @@ static:
 	cp -rfp ./result/bin/* ./bin/
 
 bin/conmon: $(OBJS) | bin
-	$(CC) $(LDFLAGS) $(CFLAGS) -o $@ $^ $(LIBS)
+	$(CC) $(LDFLAGS) $(CFLAGS) -g -o $@ $^ $(LIBS)
```

Comment:
Can it cause issues to add them every time?

Reply:
sorry, not sure what you mean, any particular case I could check for?

Reply:
do you mean anything arch-specific?

Reply:
Hm, maybe a dedicated debug build? I mean people could still strip the binary, right? So I think it should be fine to add them in general.

Reply:
Now the binary is 331KiB instead of 83KiB :), that was what I meant

Reply:
ahh, hmmmm, would you be ok with a `debug-binary` or `bin/conmon-debug` or something that has this option?

Reply:
just that i don't wanna carry a patch in the rpm source :)